### ISSUE TITLE
Error when you open vim from a different folder than elm.json is located

### ISSUE THREAD

**Issue by wellyal** [ElmCast/elm-vim#166]
I always have to `cd` for the root folder where elm.json is located so the elm-format plugin triggers correctly. Otherwise I keep receiving this error message.

```
I couldn't figure out what version of Elm you are using!

You should either run elm-format from the directory containing
your elm.json (for Elm 0.19) or elm-package.json (for Elm 0.18),
or tell me which version of Elm with --elm-version=0.19 or --elm-version=0.18
```

**Comment by Hermanverschooten** [comment]
I have the same issue, but not all the time and not in all projects.
running `:ElmFormat` works fine.

**Comment by Hermanverschooten** [comment]
The difference between the 2 projects is:
working:
* nodejs : 8.8.1
* elm-format: elm-format-0.19 0.8.0

problem:
* nodejs :  10.9.0
* elm-format: 0.8.1

**Comment by wellyal** [comment]
The problem is that the plugin is not using the latest version of elm (0.19) while the elm-webpack loader is using elm 0.19. Run `run --version` in case your elm version is updated to the latest use this [code](https://gist.github.com/wellyal/3ffef5ac6a395c56ed254878234d97af) as the ElmComponent wrapper instead of this plugin.

**Comment by ckipp01** [comment]
I may be mistaken, but I believe [this pull request](https://github.com/ElmCast/elm-vim/pull/167) should have fixed this issue no that it will fall back to FindRootDirectory. Are you able to check and report back?

**Comment by ckipp01** [comment]
@Hermanverschooten I'll look into that. I have another PR that is dealing with `elmMake` so hopefully if that gets merged in I can start working on some of the other issues with various commands, including what you just mentioned.

**Comment by Hermanverschooten** [comment]
Ok, thanks

**Comment by emarthinsen** [comment]
This would be awesome to have a fix for.

**Comment by leohaskell** [comment]
The problem was with elm-format itself, and it's already fixed there (version 0.8.2).
